smart_card_connector_app/src/application_unittest.cc

```diff
@@ -1131,4 +1132,45 @@ TEST_F(SmartCardConnectorApplicationSingleClientTest, NonExistingFunctionCall) {
   EXPECT_THAT(response, IsNullOptional());
 }
 
+// `SCardDisconnect()` and `SCardReleaseContext` calls from JS should succeed
```

Comment:
nit: Does this actually test SCardReleaseContext?

Reply:
Yes. Clarified that in a comment.